tests/validation_tests/test_pre_checks.py

```diff
@@ -0,0 +1,186 @@
+import pandas as pd
```

Comment:
could you add a test for `pre_transformation_check `please? Everything else covered but this one not actually tested.

Reply:
and test the `file_validation_failures ` look ok when raised?